frontend/src/modules/cruise-schedule/components/CruiseExportForm.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import { AppButton } from '@/core/components/AppButton';
+import { AppDropdownInput } from '@/core/components/inputs/AppDropdownInput';
+import { FileDto } from '@/cruise-applications/models/FileDto';
+import { useCruiseCsvExportMutation } from '@/cruise-schedule/hooks/CruisesApiHooks';
+import { CruiseDto } from '@/cruise-schedule/models/CruiseDto';
+
+type Props = {
+  cruises: CruiseDto[];
+  onDone?: () => void;
+};
+
```

Comment:
decide whether Props should have newline between them and their respective components (here it is, in other files there is not)